arches/install/arches-templates/project_name/templates/registration/password_reset_subject.txt

```diff
@@ -0,0 +1 @@
+Password reset for {{ app_title }}
```

Comment:
what is the use of this file?

Reply:
This seems to create the Subject Line for the password reset emails